test/test_coretasks.py

```diff
@@ -4,128 +4,111 @@
 
 import pytest
 
-from sopel import coretasks
+from sopel import config
 from sopel.module import VOICE, HALFOP, OP, ADMIN, OWNER
 from sopel.tools import Identifier
-from sopel.test_tools import MockSopel, MockSopelWrapper
-from sopel.trigger import PreTrigger, Trigger
+from sopel.tests import rawlist
 
 
 @pytest.fixture
-def sopel():
-    bot = MockSopel("Sopel")
-    return bot
+def tmpconfig(tmpdir):
+    conf_file = tmpdir.join('conf.ini')
+    conf_file.write("\n".join([
+        "[core]",
+        "owner = Uowner",
+        "nick = TestBot",
+        "enable = coretasks"
+        ""
+    ]))
+    return config.Config(conf_file.strpath)
```

Comment:
Wait a minute, why is this not using the config factory?

Reply:
A complete failure of my own doing?